lib/solid_queue/processes/poller.rb

```diff
@@ -10,7 +10,7 @@ module Poller
 
     private
       def with_polling_volume
-        if SolidQueue.silence_polling?
+        if SolidQueue.silence_polling? && ActiveRecord::Base.logger
           ActiveRecord::Base.logger.silence { yield }
```

Comment:
Maybe better just use safe-navigation `ActiveRecord::Base.logger&.silence` ?

Reply:
Then it will not `yield`

Reply:
But why? :thinking: 

```
irb(main):003> l = Logger.new(STDOUT)
irb(main):004> l.info { "OK" }
I, [2024-03-22T15:10:27.939697 #68513]  INFO -- : OK
=> true
irb(main):005> l&.info { "OK" }
I, [2024-03-22T15:10:31.957994 #68513]  INFO -- : OK
=> true


irb(main):008> n = nil
=> nil
irb(main):009> n&.info { "OK"  }
=> nil
```